assets/styles/app.scss

```diff
@@ -303,3 +399,44 @@ body {
     }
 }
 
+.load-more-tag {
+    font-family: $textFont;
+    font-size: $h6-font-size;
```

Comment:
Should be done with my custom class.
text-font
Should be done with Bootstrap.
fs-6